<launch>
    <arg name="src_image" default="/sensing/camera/traffic_light/image_raw/compressed"/>
    <arg name="scale_num" default="1"/>
    <arg name="scale_denom" default="1"/>

    <node name="demo_decode_node" pkg="turbo_decoder" exec="demo_node" output="screen">
        <remap from="src_image" to="$(var src_image)"/>
        <param name="scale_num" value="$(var scale_num)"/>
        <param name="scale_denom" value="$(var scale_denom)"/>
    </node>
</launch>
